subsys/emds/emds_flash.h

```diff
@@ -15,6 +15,19 @@
 extern "C" {
 #endif
 
+/** @brief Enumeration representing the status for a single EMDS entry
+ */
+enum emds_entry_status {
+	/** Not yet initialized */
+	EMDS_ENTRY_NO_INIT,
+	/** Valid entry */
+	EMDS_ENTRY_VALID,
+	/** Invalid old entry */
```

Comment:
Yes, it was a old entry. But not necessary invalid.

Reply:
We can discuss what is the best name. Old/Stale/Invalidated 🤷‍♂️